### DISCUSSION TITLE
Webhook Notification For HLS users playStarted and playStoped

### DISCUSSION THREAD

**Discussion by syed-MTalha** [ant-media/Ant-Media-Server#6606] [FAQs]
I have implemented the Webhook for getting the notification for my LiveApp. Webhook is sending data on users connecting through WebRTC, but when a user tried to watch a stream using HLS, webhook for not send any playStarted and playStoped notification. Can please someone help me what is the config which I am missing. on Web panel I can see both type of users watching stream on WebRTC and HLS. But no webhook for HLS users.

**Comment by nocpbxchap** [comment]
maybe you should check if status":"broadcasting means stream is active, if that is what you want to know.

**Reply by syed-MTalha** [reply]
Thanks for the response, maybe I am unable to explain it properly. For reference, I am referring the documentation here for the clarity.
https://antmedia.io/docs/guides/advanced-usage/webhooks/

I am getting actions when a user plays the stream using WebRTC, as 
**Received request** - stream_id: hJky0YoXtkx3c4Zu24632224678593, action: _**playStarted**_, timestamp: 1724355010734
**Received request** - stream_id: hJky0YoXtkx3c4Zu24632224678593, action: _**playStopped**_, timestamp: 1724355038312

But if user tries to play the same stream using HLS, I am not receiving these actions of playStarted, and playStopped. So, want to know how I can get these actions for HLS as well, or what section of config I am missing. 

Importantly I am working on Flutter, and publishing the stream using flutter app with wss, but wanted to allow users to view the stream through web using both options like WebRTC and HLS.

Thanks in advance

**Reply by Mohit-3196** [reply]
Hi @syed-MTalha,
Thank you for posting this query here.
The webhook calls are available for webrtc playback. If you would like to have it for HLS as well, we can take a **[feature request](https://github.com/ant-media/Ant-Media-Server/issues/new/choose)** for the same and assign it correct priority as per your timelines.

Can you please tell me if this is a must-have for you and if so, by when do you expect to have this feature?

Accordingly, we can discuss the next step.

Thank you.

**Reply by syed-MTalha** [reply]
Thanks @Mohit-3196 for the information. I believe this will be a good feature if we can incorporate this as it will reduce the overhead of the API calls to get the stream stats. But obviously you need to consider other factors of overhead on media server. If the overhead on the server side will not make a potential impact, I would love to request this new feature set. As working on real time socket information this will help any backend app to update the frontend without any additional overhead. I suggest it must be there

Thanks
